### ISSUE TITLE
Should static files support be a JFryServer's concern?

### ISSUE THREAD

**Issue by ggalmazor** [ggalmazor/jfry#4]
JFry is a Java web library to serve some business logic through web routes. Due to this, my initial thought on static files is that they should be the actual web container's concern.

I'm not saying that JFry shouldn't support this feature (it needs to support it). I'm saying that maybe this is configured through JFryServer implementation, not trough JFry's API.

**Comment by jkwatson** [comment]
As long as you have a way to make sure the jfry user can configure the underlying server to serve up static files, i think that it could be left out of the core API.

**Comment by ggalmazor** [comment]
Yes, that's my idea, but I'd happy to hear about some specific use cases for this to think about a simple API to be included maybe on JFryServer interface... any suggestions?
